pipe_test.go

```diff
@@ -4129,6 +4129,11 @@ func TestPipe_CleanSubscriptions_6(t *testing.T) {
 			{typ: '+', string: "punsubscribe"},
 			{typ: '_'},
 			{typ: ':', integer: 2},
+		}},
+		RedisMessage{typ: '>', values: []RedisMessage{
```

Comment:
@SoulPancake, I found this reply is incorrect. DISCARD should reply a OK message instead of a push message.

Reply:
My bad, Addressed @rueian